.travis.yml

```diff
@@ -88,6 +90,15 @@ install:
         && make
         && make install )
   - export PETSC_DIR=$PETSC_INSTALL
+  - export XSMM_INSTALL=$HOME/install/libxsmm-1.10
+  - test -s "$XSMM_INSTALL/lib/libxsmm.pc"
+        || ( curl -L https://github.com/hfp/libxsmm/archive/1.10.tar.gz > libxsmm-1.10.tar.gz
+        && mkdir libxsmm-1.10
```

Comment:
This is redundant because that is the directory name from the tarball. Why not just `tar xf libxsmm-1.10.tar.gz`?

Reply:
And presumably we have no interest in caching the tarball. Better to build outside the install directory, but if (hypothetically) there is no way to install, then still delete the tarball.